rules/integrations/gcp/credential_access_gcp_kubernetes_rolebindings_creation.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2021/06/06"
+maturity = "production"
+updated_date = "2021/06/06"
+
+[rule]
+author = ["Elastic","Austin Songer"]
+description = """
+Identifies the creation or patching of potential malicious rolebinding.
```

Comment:
How we differentiate from normal activity? If not, needs to be reformulated, also needs some context around rolebindings